backend/src/main/java/com/empire/World.java

```diff
@@ -398,33 +398,28 @@ public static World startNew(String gmPasswordHash, String obsPasswordHash, Map<
 				if (i == 0) {
 					c.addTag("Ruler");
 					if ("checked".equals(setup.food)) c.values.add("food");
-					if ("checked".equals(setup.prosperity)) c.values.add("prosperity");
 					if ("checked".equals(setup.happiness)) c.values.add("happiness");
 					if ("checked".equals(setup.supremacy)) c.values.add("supremacy");
-					if ("checked".equals(setup.conquest)) c.values.add("conquest");
+					if ("checked".equals(setup.conquest)) c.values.add("territory");
 					if ("checked".equals(setup.glory)) c.values.add("glory");
 					if ("checked".equals(setup.religion)) c.values.add("religion");
 					if ("checked".equals(setup.ideology)) c.values.add("ideology");
 					if ("checked".equals(setup.security)) c.values.add("security");
 					if ("checked".equals(setup.riches)) c.values.add("riches");
 					if ("checked".equals(setup.friendship)) c.values.add("friendship");
 					if ("checked".equals(setup.culture)) c.values.add("culture");
-					if ("checked".equals(setup.unity)) c.values.add("unity");
-					if ("checked".equals(setup.worldpeace)) c.values.add("worldpeace");
 				} else {
 					c.values.add("food");
-					c.values.add("prosperity");
 					if (Math.random() < 0.1) c.values.add("happiness");
 					if (Math.random() < 0.1) c.values.add("supremacy");
```

Comment:
Should this line also be removed now that supremacy has been nixed?

Reply:
Done.